windows-gaming-driver/src/driver/controller.rs

```diff
@@ -308,6 +330,10 @@ impl Controller {
         assert!(self.ga != State::Suspending, "trying to exit from a suspending vm?");
         assert!(self.ga != State::Suspended, "trying to exit from a suspended vm?");
         self.ask_clipboard();
+        // if we have a temporary entry, get rid of it
+        if let Some(w) = mem::replace(&mut self.temporary_entry_writer, None) {
+            drop(w);
```

Comment:
I'm not sure if your code drops the writer instantaneously or if it's dropped at the end of the function. I want it to be dropped immediately before performing anything after.

Actually due to tokio, there is no way that this would make the semantics any different… So I guess it's fine in your way (which I used first before thinking about it).